llvm/test/ObjectYAML/DXContainer/RootSignature-Flags.yaml

```diff
@@ -25,7 +25,7 @@ Parts:
 # CHECK-NEXT:    RootSignature:
 # CHECK-NEXT:      Version: 2
 # CHECK-NEXT:      NumStaticSamplers: 0
-# CHECK-NEXT:      StaticSamplersOffset: 24
+# CHECK-NEXT:      StaticSamplersOffset: 0
```

Comment:
This suggests that this is a functional change.  Maybe at least mention in the PR description / commit message why it's expected that these values have changed?  Or maybe it's actually desireable that we fix it so that these get set to a value?